src/Resie.jl

```diff
@@ -62,69 +62,91 @@ function run_simulation(project_config::Dict{AbstractString,Any})
     )
     EnergySystems.set_timestep(parameters["time_step_seconds"])
 
-    outputs = output_keys(components, project_config["io_settings"]["output_keys"])
-    reset_file(project_config["io_settings"]["output_file"], outputs)
-
-    ### set data for profile line plot
-    # check if profile line plot should be created (checks if output_plot is present or if
-    # it is "nothing")
-    plot_bool = !(
-        !(haskey(project_config["io_settings"], "output_plot"))
-        ||
-        project_config["io_settings"]["output_plot"] === "nothing"
-    )
+    # read in flags, if all outputs should be printed and/or plotted
+    plot_all_outputs_bool = false
```

Comment:
Small nitpick, but generally variable names should not (need to) contain their type. For boolean variables one convention is to use a "is_", has_" or "do_" prefix.

Reply:
Your right, I will change that!